<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialogWindow</class>
 <widget class="QDialog" name="MyDialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>288</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>383</width>
    <height>288</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>383</width>
    <height>288</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/MacLookUp.png</normaloff>:/MacLookUp.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Background.png);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>MAC adress:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="Search_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&amp;Search</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Close_button">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&amp;Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>Searching... %p%</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>64</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logo.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <action name="img">
   <property name="icon">
    <iconset>
     <normalon>MacLookUp.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>OfflineMacFinder</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>Search_button</tabstop>
  <tabstop>Close_button</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Close_button</sender>
   <signal>clicked()</signal>
   <receiver>MyDialogWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
